### ISSUE TITLE
Possible bug in 2d benchmark tutorial, case 3

### ISSUE THREAD

**Issue by jhabriel** [pmgbergen/porepy#1095]
In the tutorial, the aperture is specified as a material (solid) constant employing the keyword `aperture`. I might be wrong, but could it be that the right keyword should be `residual_aperture` instead? (See notebook cell 9, second line).

**Comment by IvarStefansson** [comment]
I believe you are right. This risk of passing wrong or misspelled constants to MaterialConstants has actually worried and bothered me a bit. Do you think it would make sense to do some safeguarding in [init](https://github.com/pmgbergen/porepy/blob/f2a532f0056fbc0e2e51eb39c2992fcf4f1c18e1/src/porepy/models/material_constants.py#L164)? What about you, @keileg?

**Comment by IvarStefansson** [comment]
The safeguard would be some kind of check that the passed constants are all expected/valid for the MaterialConstants in question.

**Comment by jhabriel** [comment]
Yes, I believe some level of safeguarding is needed, because these are bugs that are really difficult to spot.

Perhaps, a warning can be raised when the user specifies an unsupported keyword in the declaration of the material constants.

**Comment by keileg** [comment]
Agree, safeguarding sounds like a good idea

**Comment by IvarStefansson** [comment]
Something like this, then?
```
for key in user_dict:
   if key not in default_dict:
      raise ValueError("...")
```
